### ISSUE TITLE
STM test failure

### ISSUE THREAD

**Issue by jdegoes** [zio/zio#1211]
```
[error]              x compute a `Ref` from 2 variables, increment the first `Ref` and decrement the second `Ref` in different fibers.
[error]  10000 == '10000' (STMSpec.scala:236)
```

**Comment by LGLO** [comment]
There are two test cases (e17, e18) that try to prove that implementation with `Ref` will always yield wrong results. It happened that it didn't.
I think we should not keep these two.

**Comment by jdegoes** [comment]
Let's delete them! 👍

**Comment by LGLO** [comment]
#1215 :)

**Comment by jdegoes** [comment]
Fixed by #1215
